docs/JMAQS_1/appium/AppiumTestObject.md

```diff
@@ -1,84 +1,26 @@
-# <img src="resources/maqslogo.ico" height="32" width="32"> AppiumTestObject
-Appium test context data
+# <img src="resources/jmaqslogo.jpg" height="32" width="32"> Appium Test Object
 
-## Inheritance Hierarchy
-```java
-BaseTestObject
-    com.magenic.jmaqs.appium.AppiumTestObject
-```
-Package: com.magenic.jmaqs.appium;  
-Assembly: import com.magenic.jmaqs.appium.AppiumTestObject
-
-## Syntax
-Java
-```java
-public class AppiumTestObject
-```
+## Overview
+Takes care of the base test context data.
 
-The AppiumTestObject type exposes the following members.
+[GetAppiumDriver](#GetAppiumDriver)  
+[SetAppiumDriver](#SetAppiumDriver)  
+[GetAppiumManager](#GetAppiumManager)     
 
-## Constructors
-[AppiumTestObject](#AppiumTestObject)  -  Initializes a new instance of the AppiumTestObject class
-
-## Properties
-[AppiumDriver]() - Gets the Appium driver  
-[AppiumManager]() - Gets the Appium driver manager    
-[Log]() - (Inherited from BaseTestObject.)  
-[ManagerStore]() - (Inherited from BaseTestObject.)  
-[Objects]() - (Inherited from BaseTestObject.)  
-[PerfTimerCollection]() - (Inherited from BaseTestObject.)  
-[Values]()  - (Inherited from BaseTestObject.)  
-
-## Methods 
-[GetAppiumDriver](#GetAppiumDriver)  -  Gets the Appium driver  
-[GetAppiumManager](#GetAppiumManager)  - Gets the Appium manager  
-[SetAppiumDriver](#SetAppiumDriver)  -  Sets the Appium driver
-
-##  AppiumTestObject
-Instantiates a new Appium test object.
-#### Written as
- ```java
-public AppiumTestObject(AppiumDriver<WebElement> appiumDriver, Logger logger, String fullyQualifiedTestName)
-```
-#### Example
+## GetAppiumDriver
+Gets the web service driver
 ```java
-AppiumDriver<WebElement> defaultMobileDriver = AppiumDriverFactory.getDefaultMobileDriver();
-AppiumTestObject appiumTestObject = new AppiumTestObject(defaultMobileDriver,
-new ConsoleLogger(), method.getName());
+AppiumDriver<WebElement> driver = this.AppiumManager.getAppiumDriver();
 ```
 
-##  GetAppiumDriver
-Gets the Appium Driver
-#### Written as
+## SetAppiumDriver
+Sets the web service driver
```

Comment:
webservices reference

Reply:
webservices reference has been deleted and replaced with Appium